terminal/terminal.go

```diff
@@ -166,3 +182,68 @@ func getConfigFilePath(file string) (string, error) {
 	}
 	return path.Join(usr.HomeDir, configDir, file), nil
 }
+
+func loadConfig(cmds *Commands) *config {
+	fullConfigFile, err := getConfigFilePath(configFile)
+	if err != nil {
+		fmt.Printf("Unable to load config file: %v.", err)
+	}
+
+	f, err := os.Open(fullConfigFile)
+	if err != nil {
+		f, err = os.Create(fullConfigFile)
+		if err != nil {
+			fmt.Printf("Unable to create config file: %v.", err)
+		} else {
+			err = writeDefaultConfig(f, cmds)
```

Comment:
That makes sense. No reason to overcomplicate it since we're not loading anything.